.github/workflows/build.yml

```diff
@@ -142,7 +142,8 @@ jobs:
       id: platform
       run: |
         platform=${{ matrix.os }}
-        platform=${platform/windows-*/windows-latest}
+        platform=${platform/windows-2019/windows-latest}
+        platform=${platform/%-arm/-arm64}
```

Comment:
We should name the new one `windows-arm64`, not `windows-11-arm64` see https://github.com/ruby/setup-ruby/pull/750#discussion_r2059091375